[docs] Learn Material-UI (#13624)
mui-org/material-ui: docs/src/pages/getting-started/learn/learn.md

# Learn Material-UI

<p class="description">New to Material-UI? It's easy to learn if you know where to start! This guide will help you to get started quickly.</p>

People come to Material-UI from different backgrounds and with different learning styles. Whether you prefer a more theoretical or a practical approach, we hope you’ll find this section helpful.

> Like any unfamiliar technology, Material-UI does have a learning curve. With practice and some patience, you will get the hang of it.

Reviewer: ```suggestion 
Like any unfamiliar technology, Material-UI does have a learning curve. With practice, and some patience, you will soon get the hang of it.
```

## First Example

The [usage page](/getting-started/usage/#quick-start) contains a small Material-UI example with a live editor.
Even if you don’t know anything about Material-UI yet, try changing the code and see how it affects the result.

## Example Projects

We [host some example projects](/getting-started/example-projects/) providing the infrastructure needed to develop and deploy websites in React.

## Page layout examples

We have a [selection of basic page layouts](/getting-started/page-layout-examples/) to help you get started building your app.

## Recommended resources

> Sometimes people find third-party blog posts, books and video courses more helpful than the official documentation. We maintain a list of commonly recommended resources, some of which are free.

Reviewer: ```suggestion
When first learning Material-UI, you might find third-party blog posts, books, and video courses more helpful than the official documentation. Here are some recommended resources, many of which are free.
```

### Free

- **Meet Material-UI — your new favorite user interface library**: a blog post that guides you in building a Todo MVC while covering some important concepts of Material-UI.

>  - 📝 [The blog post](https://medium.freecodecamp.org/meet-your-material-ui-your-new-favorite-user-interface-library-6349a1c88a8c)

Reviewer: ```suggestion
 - 📝 [The blog post](https://medium.freecodecamp.org/meet-your-material-ui-your-new-favorite-user-interface-library-6349a1c88a8c)
 - 📹 [The videos](https://www.youtube.com/watch?list=PLcCp4mjO-z98WAu4sd0eVha1g-NMfzHZk&v=xm4LX5fJKZ8)
```

> - **Learn React & Material-UI**: a series of videos covering all the important Material-UI's components.

Reviewer: ```suggestion
- **Learn React & Material-UI**: a series of videos covering all the important Material-UI components.
````

  - 📹 [The videos](https://www.youtube.com/watch?v=xm4LX5fJKZ8&list=PLcCp4mjO-z98WAu4sd0eVha1g-NMfzHZk)
- **Getting Started With Material-UI For React**: a blog post that guides you in building a simple card list.
  - 📝 [The blog post](https://medium.com/codingthesmartway-com-blog/getting-started-with-material-ui-for-react-material-design-for-react-364b2688b555)
  - 📹 [The video](https://www.youtube.com/watch?v=PWadEeOuv5o)

> - **Elegant UX in React with Material-UI**: a blog post covering some important concepts of Material-UI.

Reviewer: ```suggestion
- **Elegant UX in React with Material-UI**: a blog post covering some important Material-UI concepts.
```
